src/content-script.ts

```diff
@@ -4,14 +4,47 @@ import Mustache from "mustache";
 import { createElement } from "./elements";
 import { isBodyWithReplies, isRepositoryDetails } from "./validations";
 
-(async function main() {
+const regexPathName = /\/.+\/.+\/(issues|pull)\/\d+/;
```

Comment:
[Naming] I'm not a fan of including the data type of a variable in the variable's name. We already know what it is. Maybe...

```suggestion
const pathNameMatcher = /\/.+\/.+\/(issues|pull)\/\d+/;
```

_(this is not a strong request, if you hate what I'm saying it's certainly not a blocker 😄 )_